subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -125,3 +126,89 @@ def build_lts_payloads() -> dict:
             "$schema": "urn:rhods-matbench-upload:2.0.0",
             **_convert_steps(data)
         }, start_time, end_time
+
+
+def build_limited_lts_payload() -> dict:
```

Comment:
I guess this can replace `build_lts_payload`, we'll want only one version

Reply:
can you modify `MatrixBenchmarking` to have a `pretty` printing of the Json? 
I though it was there, but the file [lts_payload.json](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/689/pull-ci-openshift-psap-ci-artifacts-main-ods-notebooks-light/1649131612962557952/artifacts/notebooks-light/test/artifacts/003__plots/lts_payload.json) isn't pretty printed :/